telemetry/definitions/commonDefinitions.json

```diff
@@ -2771,6 +2771,18 @@
                 { "type": "amazonqGenerateApproachLatency" }
             ]
         },
+        {
+            "name": "amazonq_startConversationInvoke",
+            "description": "Captures startConversation process",
+            "metadata": [{ "type": "amazonqConversationId", "required": false }]
+        },
+        {
+            "name": "amazonq_createUploadUrl",
+            "description": "Captures createUploadUrl process",
+            "metadata": [
+                { "type": "amazonqConversationId" }
```

Comment:
Do we also want to capture how long create upload url took?

Reply:
Do we? I think that all performance metrics specific to the api are tracked already in the backend operations themselves. toolkit performance metrics should be based in other concepts rather than api latency.

Reply:
I'm going to add though repository size for uploadCode though and converastionId in the startConversation (if succeeded)

Reply:
It's not really relevant in the scope of this task though